low-level/files/opt/seagate/sspl/setup/resource_map/lr2/storage.py

```diff
@@ -109,6 +109,36 @@ def get_health_info(self, rpath):
                 errno.EINVAL, msg)
         return info
 
+    @staticmethod
+    def get_fan_specfics(fan):
+        return {
+            "uid": fan.get("durable-id", "NA"),
+            "location": fan.get("location", "NA"),
+            "status": fan.get("status", "NA"),
+            "speed": fan.get("speed", "NA"),
+            "serial-number": fan.get("serial-number", "N/A"),
+            "part-number": fan.get("part-number", "N/A"),
+            "health": fan.get("health", "OK"),
+        }
+
+    def get_fanmodules_info(self):
+        response = []
+        fanmoduels_data = self.get_realstor_encl_data("fan-modules")
```

Comment:
Can change the var name to fanmodules_data. I think it's a typo error #L126

Reply:
Done.